src/main/scala/chisel3/aop/injecting/InjectingAspect.scala

```diff
@@ -42,17 +42,23 @@ abstract class InjectorAspect[T <: RawModule, M <: RawModule](
     injection: M => Unit
 ) extends Aspect[T] {
   final def toAnnotation(top: T): AnnotationSeq = {
-    toAnnotation(selectRoots(top), top.name)
+    val moduleNames = Select.collectDeep(top) { case i => i.name }.toSeq
+    toAnnotation(selectRoots(top), top.name, moduleNames)
   }
 
-  final def toAnnotation(modules: Iterable[M], circuit: String): AnnotationSeq = {
+  final def toAnnotation(modules: Iterable[M], circuit: String, moduleNames: Seq[String]): AnnotationSeq = {
```

Comment:
Scaladoc for this might be useful? Is there supposed to be some parallel structure between modules and moduleNames?